GitVersionCore/Configuration/ConfigSerialiser.cs

```diff
@@ -17,13 +17,22 @@ public static Config Read(TextReader reader)
             return deserialize;
         }
 
+        public static void Write(Config config, TextWriter writer)
+        {
+            var serializer = new Serializer(SerializationOptions.None, new HyphenatedNamingConvention());
+            serializer.Serialize(writer, config);
+        }
+
         public static void WriteSample(TextWriter writer)
         {
             writer.WriteLine("# assembly-versioning-scheme: MajorMinorPatchMetadata | MajorMinorPatch | MajorMinor | Major");
-            writer.WriteLine("# develop-branch-tag: alpha");
-            writer.WriteLine("# release-branch-tag: rc");
             writer.WriteLine("# tag-prefix: '[vV|version-] # regex to match git tag prefix");
             writer.WriteLine("# next-version: 1.0.0");
+            writer.WriteLine("# mode: ContinuousDelivery | ContinuousDeployment");
+            writer.WriteLine("# continuous-delivery-fallback-tag: ci");
+            writer.WriteLine("#branches:");
+            writer.WriteLine("#   release[/-]*:\n    mode: ContinuousDelivery | ContinuousDeployment\n    tag: rc");
```

Comment:
Again, nice spot. Fixed in https://github.com/JakeGinnivan/GitVersion/commit/e549a4d81b2a08f4ec4baf28b1be700eb13ed0d4